api/sft/service.go

```diff
@@ -21,13 +23,13 @@ type Service struct {
 // based on srt, takes a db, context, and pool, all of which that are created in the calling application, returns a service
 func NewService(db ISimpleFeatureToggleDb, ctx context.Context, pool *pgxpool.Pool) *Service {
 	fmt.Println("new service!")
-	if err := RunDbMigrations(ctx, pool); err != nil {
-		panic(err)
-	}
-
-	if err := RunDbSeed(ctx, pool); err != nil {
-		log.Println("db seed failed")
-	}
+	//if err := RunDbMigrations(ctx, pool); err != nil {
```

Comment:
Commented code can either be used or removed - tend to prefer to use git to reclaim code vs comments.

Reply:
Removed.